dts/bindings/spi/sifive,spi0.yaml

```diff
@@ -5,8 +5,14 @@ description: Sifive SPI controller
 
 compatible: "sifive,spi0"
 
-include: spi-controller.yaml
+include: [spi-controller.yaml, pinctrl-device.yaml]
 
 properties:
     reg:
       required: true
+
+    pinctrl-0:
+      required: false
+
+    pinctrl-names:
+      required: false
```

Comment:
this is redundant, none are required by default. applies to others as well.

Reply:
Removed redundant fields from binding yamls